.travis.yml

```diff
@@ -4,58 +4,42 @@ branches:
 os: osx
 matrix:
   include:
-    # Note: The ordering of the environments is optimized so that the complete matrix runs as quickly as possible,
-    # while also providing the first results as early as possible.
-    # At the moment, build times for each environment are roughly as follows:
-    # xcode7.3: 16 minutes
-    # xcode8.3: 7 minutes
-    # xcode9.2: 7 minutes
-    # xcode10.1: 6 minutes
-    # https://docs.travis-ci.com/user/reference/osx/#macos-version
-    # This data is based on the following build, which should be similar to most builds on the repo:
-    # https://travis-ci.org/pybee/rubicon-objc/builds/506412489
-    # Currently, Travis allows a maximum of two concurrent Mac builds per owner:
-    # https://blog.travis-ci.com/2017-09-22-macos-update
-    # With the following order, xcode7.3 will take up the first slot, and at the same time the three other
-    # environments use the second slot in sequence. xcode10.1 will finish first and provide some early results - most
-    # test failures happen in all environments and will be quickly reported this way.
-
     # OSX 10.11 El Capitan
     - osx_image: xcode7.3
+    # macOS 10.12 Sierra
+    - osx_image: xcode8.3
+    - osx_image: xcode9.2
     # macOS 10.13 High Sierra
     - osx_image: xcode10.1
     # macOS 10.14 Mojave
     - osx_image: xcode10.2
-    # macOS 10.12 Sierra
-    - osx_image: xcode9.2
-    - osx_image: xcode8.3
 language: generic
 cache:
   directories:
     ~/.pyenv
-env:
-  global:
-    - PY35="3.5.6"
-    - PY36="3.6.8"
-    - PY37="3.7.2"
 install:
-  # The different Travis environments have different pyenv versions, and not all have the Python versions we need,
-  # so we upgrade pyenv to get a consistent setup in all environments.
-  - brew update
-  - brew outdated pyenv || brew upgrade pyenv
-  # The existing pyenv shims from the build cache may no longer be valid, for example if pyenv was upgraded
-  # (in which case the paths used in the shims point to an older, no longer installed pyenv version).
-  # By rehashing the shims we can be sure that they are always valid again.
-  - pyenv rehash
   # pyenv's shims directory is not in the PATH by default.
   - export PATH="$(pyenv root)/shims:${PATH}"
-  # These are the Python versions that we want to test on.
-  - pyenv install --skip-existing ${PY35}
-  - pyenv install --skip-existing ${PY36}
-  - pyenv install --skip-existing ${PY37}
-  - pyenv global ${PY37} ${PY36} ${PY35} system
-  - pip install --upgrade tox
-  - make -f Makefile
+  # Diagnostic: Find out which versions of Python are availalbe.
+  - pyenv install -l
```

Comment:
I know it's not required, but I left it in because it costs almost nothing to run, but it saves us having to introduce the same line in the future if Travis ever changes their deployment configuration (or when we add/remove a macOS version and want to re-assess our available versions)

Reply:
That makes sense, I'm fine with that - just wanted to be sure that it was intentional.